package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "telemetry-events",
-  "version": "1.0.1",
+  "version": "1.1.0",
```

Comment:
I just...didn't understand the version change. ...trying to... stay synced on versioning, but I won't keep you.